pkg/virt-operator/creation/components/crds.go

```diff
@@ -488,14 +488,14 @@ func NewPrometheusRuleSpec(ns string) *promv1.PrometheusRuleSpec {
 				Name: "kubevirt.rules",
 				Rules: []promv1.Rule{
 					{
-						Record: "num_of_running_virt_api_servers",
+						Record: "kubevirt_up_virt_api_servers_total",
 						Expr: intstr.FromString(
 							fmt.Sprintf("sum(up{namespace='%s', pod=~'virt-api-.*'})", ns),
 						),
 					},
 					{
 						Alert: "VirtAPIDown",
-						Expr:  intstr.FromString("num_of_running_virt_api_servers == 0"),
+						Expr:  intstr.FromString("kubevirt_up_virt_api_servers_total == 0"),
```

Comment:
This is a bigger rename

Reply:
This record is used by the VirtAPIDown and for the LowVirtAPICount alerts, we change it so it will be associated with kubevirt (this is true for the entire PR), since most metric/rule/alert names are prefixed by the component that ships it.

Reply:
I did not really understand the reasoning for the name, nor why plain prepending was not enough. My point was that this rename was not mentioned and explained in the PR text. It may even deserve a commit of its own - I don't know.

Reply:
It's part of the PR text now, and following a convention for metrics afaiu, @sradco can probably help here as well :)